x/auth/simulation/genesis.go

```diff
@@ -120,3 +166,20 @@ func RandomizedGenState(simState *module.SimulationState, randGenAccountsFn type
 	fmt.Printf("Selected randomly generated auth parameters:\n%s\n", bz)
 	simState.GenState[types.ModuleName] = simState.Cdc.MustMarshalJSON(authGenesis)
 }
+
+// RandomizedGenState2 generates a random GenesisState for auth
+func RandomizedGenState2(r *rand.Rand, genState map[string]json.RawMessage, cdc codec.JSONCodec, bondDenom string, accs []simtypes.Account, initialStake sdkmath.Int, numBonded int, genTimestamp time.Time) {
```

Comment:
rng or prng

Reply:
I prefer to leave these. The existing simulator uses `r` everywhere, and these functions are just translations not new code.